scripts/init.js

```diff
@@ -96,15 +96,24 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
     console.log('Success! Created ' + appName + ' at ' + appPath + '.');
     console.log('Inside that directory, you can run several commands:');
     console.log();
-    console.log('  * npm start: Starts the development server.');
-    console.log('  * npm test: Starts the test runner.');
-    console.log('  * npm run build: Bundles the app into static files for production.');
-    console.log('  * npm run eject: Removes this tool. If you do this, you can’t go back!');
+    console.log(chalk.cyan('npm start'));
+    console.log('  Starts the development server.');
+    console.log();
+    console.log(chalk.cyan('npm test'));
+    console.log('  Starts the test runner.');
+    console.log();
+    console.log(chalk.cyan('npm run build'));
+    console.log('  Bundles the app into static files for production.');
+    console.log();
+    console.log(chalk.cyan('npm run eject'));
+    console.log('  Removes this tool and copies build dependencies, configs,')
+    console.log('  and scripts into the app directory.')
```

Comment:
Looks like this line and the one above it are missing semicolons in the end, can you add them back? Otherwise everything seems to be in order now 👌